packages/google_sign_in/google_sign_in_web/README.md

```diff
@@ -96,7 +96,7 @@ See the [google_sign_in.dart](https://github.com/flutter/plugins/blob/master/pac
 
 ## Contributions and Testing
 
-Tests are a crucial to contributions to this package. All new contributions should be reasonably tested.
+Tests are crucial to perform before contributing to this package. All new contributions should be reasonably tested.
```

Comment:
This is changing the meaning; the intent is not just that people need to run tests, but also add tests.

Just removing "a" would fix the grammar without changing the meaning.